crates/rerun_c/src/lib.rs

```diff
@@ -269,6 +275,12 @@ fn rr_recording_stream_new_impl(
         .store_source(re_log_types::StoreSource::CSdk)
         .default_enabled(default_enabled);
 
+    if !(recording_id.is_null() || recording_id.is_empty()) {
```

Comment:
nit: afaik in another place we recently decided to only check is_empty, but can't hurt 🤷